data-prepper-expression/src/main/antlr/DataPrepperExpression.g4

```diff
@@ -181,6 +186,13 @@ literal
     | Null
     ;
 
+fragment
+SetLiteral
```

Comment:
I don't think we want another fragment here. We should use the other `literal` and remove this.

Reply:
I felt doing something like `/boolean_value in {true, false}` doesn't make sense. Similarly, I felt checking for `null` in a set is not that useful

Reply:
Maybe it's not very useful, but the literals may grow as well. I think if we just have a literal, we can have a nice recursive structure. This is also similar to other programming languages.